### PR TITLE
Edit utility classes test example

### PR DESCRIPTION
**Pull request by fermujica**

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 * xref:munit-cookbook.adoc[MUnit Examples]
   ** xref:mock-before-after-foreach-cookbook.adoc[Mocking a Message Before and Inside a Foreach Processor]
   ** xref:test-mock-errors-cookbook.adoc[Testing and Mocking Errors]
-  ** xref:utility-classes-dw-cookbook.adoc[Using Utility Classes With a DataWeave Expression From a Test]
+  ** xref:utility-classes-cookbook.adoc[Using Utility Classes From a Test]
```
Comment (kbostrom) [inline modules/ROOT/nav.adoc]:
```suggestion
  ** xref:utility-classes-cookbook.adoc[Using Utility Classes from a Test]
```

File: modules/ROOT/pages/utility-classes-cookbook.adoc [hunk 2]
```diff
@@ -21,9 +21,8 @@ public class Helper {
 }
 ----
 
-[CAUTION]
-====
-The package of the class needs to be exported in the `mule-artifact.json` file:
+When using custom classes, the package of the class must be exported in the `mule-artifact.json` file:
```
Comment (kbostrom) [inline modules/ROOT/pages/utility-classes-cookbook.adoc:8]: Is this something that your audience should know BEFORE they see the example above; is it a prerequisite to the above syntax example?

File: modules/ROOT/pages/utility-classes-cookbook.adoc [hunk 3]
```diff
@@ -38,10 +37,8 @@ The package of the class needs to be exported in the `mule-artifact.json` file:
   }
 }
 ----
-====
-
 
-== Using DataWeave
+== Invoking your Java Class Using DataWeave
```
Comment (kbostrom) [inline modules/ROOT/pages/utility-classes-cookbook.adoc]: Note that the intro text mentions Java Module before DataWeave, but the related subtopics are in the opposite order. Suggest reordering one or the other.

File: modules/ROOT/pages/utility-classes-cookbook.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@ endif::[]
 
 When developing test cases, you might want to use Java classes located in `src/test/java`. You can use Java classes through Java Module or through DataWeave.
```
Comment (kbostrom) [inline modules/ROOT/pages/utility-classes-cookbook.adoc]: Is there anything to add, either here or in the related subtopic intros, about when a user might choose Java Module and when DataWeave?

File: modules/ROOT/pages/utility-classes-cookbook.adoc (hunk 3 above)
Comment (kbostrom) [inline modules/ROOT/pages/utility-classes-cookbook.adoc]:
```suggestion
== Invoking Your Java Class Using DataWeave
```

File: modules/ROOT/pages/utility-classes-cookbook.adoc [hunk 5]
```diff
@@ -58,7 +55,7 @@ To invoke static methods using DataWeave you must add the `java!` prefix to the
 </munit:test>
 ----
 
-== Using Java Module
+== Invoking your Java Class Using Java Module
```
Comment (kbostrom) [inline modules/ROOT/pages/utility-classes-cookbook.adoc]:
```suggestion
== Invoking Your Java Class Using Java Module
```
Tip: You can check title capitalization by going to this website: https://capitalizemytitle.com/

Comment (kbostrom) [reply modules/ROOT/pages/utility-classes-cookbook.adoc]: Can we provide a sentence transition between the subtitle and the first step?

File: modules/ROOT/pages/utility-classes-cookbook.adoc [hunk 6]
```diff
@@ -58,7 +55,7 @@ To invoke static methods using DataWeave you must add the `java!` prefix to the
...
 
 . Ensure that the Java module is in your project:
```
Comment (kbostrom) [inline modules/ROOT/pages/utility-classes-cookbook.adoc]: Is Java Module the name of a thing, or is a/the Java module the thing itself?
Init. Caps. for a proprietary, owned name; lowercase "module" for any module of type Java.
Search for and standardize throughout this text.

Comment (kbostrom) [reply modules/ROOT/pages/utility-classes-cookbook.adoc]: Based on the previous comment, STET the sentence above
or
```suggestion
. Ensure that Java Module is in your project:
```

File: modules/ROOT/pages/utility-classes-cookbook.adoc [hunk 7]
```diff
@@ -5,7 +5,7 @@ endif::[]
 
 When developing test cases, you might want to use Java classes located in `src/test/java`. You can use Java classes through Java Module or through DataWeave.
 
-For the following example, assume that your class is located in `src/test/java/com/example/MyUtilClass.java`:
+For example, assume that you want to use in an MUnit test a `MyUtilClass.java` class located in `src/test/java/com/example/`:
 
 .MyUtilClass
```
Comment (kbostrom) [inline modules/ROOT/pages/utility-classes-cookbook.adoc]: Is it necessary or is it just common practice to include identifiers to code snippets?
I don't see everyone doing this, but that does not mean that it's wrong.
I will edit them as if they were titles or headings.

File: modules/ROOT/pages/utility-classes-cookbook.adoc [hunk 8]
```diff
@@ -38,10 +37,8 @@ The package of the class needs to be exported in the `mule-artifact.json` file:
   }
 }
 ----
-====
-
 
-== Using DataWeave
+== Invoking your Java Class Using DataWeave
 
 To invoke static methods using DataWeave you must add the `java!` prefix to the package name of the class:
```
Comment (kbostrom) [inline modules/ROOT/pages/utility-classes-cookbook.adoc:44]: In Line 45, lowercase "class."

File: modules/ROOT/pages/utility-classes-cookbook.adoc [hunk 9]
```diff
@@ -74,7 +71,7 @@ To invoke static methods using DataWeave you must add the `java!` prefix to the
 </dependency>
 ----
 . Use any of the Java Module's operations. +
-In this case, invoke a static method of a class:
+For this example, invoke a static method of a class:
 +
 .Using a class with the Java Module
```
Comment (kbostrom) [inline modules/ROOT/pages/utility-classes-cookbook.adoc:78]: If Java Module is a proper noun, remove "the."
If not, change to "the Java module."

Comment (kbostrom) [reply modules/ROOT/pages/utility-classes-cookbook.adoc:78]: I've looked this up on the internet many times, and been in discussions with George and PMMs. No one seems to agree. Both uses are common. So if you have any advice or clarity about this, I'd sure love to hear it!

**Review by kbostrom — APPROVED** [review]
Here you go!

File: modules/ROOT/pages/utility-classes-cookbook.adoc (hunk 5 above)
Comment (fermujica) [reply modules/ROOT/pages/utility-classes-cookbook.adoc]: Good idea. Done.

**Pull request merged**